ui/apps/platform/cypress/helpers/main.js

```diff
@@ -56,6 +57,8 @@ const routeMatcherMap = {
     ...routeMatcherMapForComplianceLevelsByStandard,
 };
 
+const routeMatcherMapForConsole = undefined;
```

Comment:
TBD - we can decide which console requests are mandatory to wait on in the future and update in this single location.